plugins/obs-qsv11/obs-qsv11.c

```diff
@@ -566,13 +624,102 @@ static void update_params(struct obs_qsv *obsqsv, obs_data_t *settings)
 	info("debug info:");
 }
 
+static void update_params_av1(struct obs_qsv *obsqsv, obs_data_t *settings)
+{
+	video_t *video = obs_encoder_video(obsqsv->encoder);
+	const struct video_output_info *voi = video_output_get_info(video);
+
+	const char *target_usage =
+		obs_data_get_string(settings, "target_usage");
+	const char *profile = obs_data_get_string(settings, "profile");
+	const char *rate_control =
+		obs_data_get_string(settings, "rate_control");
+	int target_bitrate = (int)obs_data_get_int(settings, "bitrate");
+	int max_bitrate = (int)obs_data_get_int(settings, "max_bitrate");
+	int keyint_sec = (int)obs_data_get_int(settings, "keyint_sec");
+	bool cbr_override = obs_data_get_bool(settings, "cbr");
```

Comment:
This was an override for a setting that was replaced over seven years ago. I don't think we need to carry this over to a brand new encoder, since `settings` are encoder specific.